code/game/objects/items/explosives/plastique.dm

```diff
@@ -32,7 +32,10 @@
 		icon_state = "[icon_state][alarm_sounded ? "_armed" : "_on"]"
 
 /obj/item/explosive/plastique/attack_self(mob/user)
+	. = ..()
 	var/newtime = tgui_input_number(usr, "Please set the timer.", "Timer", 10, 60, 10)
+	if(!newtime)
+		return
```

Comment:
```suggestion
	if(newtime < 1)
		return
```

Reply:
Should I not make it where it checks the minimum and maximum amount of the timer instead?

Reply:
the tgui_input_number clamps it between 10 and 60, so !newtime is fine to me, since this only comes up if nothing is entered at all.

Reply:
That was my reasoning for it as well